apps/api/src/server/repositories/representation.repository.js

```diff
@@ -126,9 +126,23 @@ export const getById = async (id, caseId) => {
 			},
 			representationActions: {
```

Comment:
@Philip-Philippou1 do we want to change this in the schema to workflow ?

Reply:
I think this is ok, representationActions is more descriptive in the DB and describes where the table belongs.